### PR TITLE
Modified copiright and add AudioConvert and criterian classes

### PR DESCRIPTION
**Pull request by LimbertVargas**
Hello mates, please review the implementations. 
Regards

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    public int getaRate() {
+        return aRate;
+    }
+
+    public void setaRate(int aRate) {
+        this.aRate = aRate;
+    }
+}
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java]: Please add a space.

File: src/main/java/com/fundation/webservice/model/ConvertPptToImage.java [hunk 2]
```diff
@@ -24,7 +22,7 @@
  * @author Limbert Alvaro Vargas Laura
  * @version 1.0
  */
-public class ConvertPptToImage {
+/*public class ConvertPptToImage {
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/fundation/webservice/model/ConvertPptToImage.java]: Limbert, there is commented code

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 1 above)
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/CriteriaAudio.java]: Please leave an extra final blank line

Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java]: fixed

File: src/main/java/com/fundation/webservice/model/ConvertPptToImage.java (hunk 2 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/ConvertPptToImage.java]: yes my friend, I fixed it

File: src/main/java/com/fundation/webservice/model/CriteriaAudio.java (hunk 1 above)
Comment (LimbertVargas) [reply src/main/java/com/fundation/webservice/model/CriteriaAudio.java]: ok alejandro, I fixed it

**Review by JosueRodriguezGarcia — COMMENTED** [review]
Ok, Limbert great job.

**Pull request merged**